terraform/cluster/aws/main.tf

```diff
@@ -150,14 +150,21 @@ module "ebs_csi_driver_controller" {
     null_resource.wait_k8s_api
   ]
 
-  source  = "DrFaust92/ebs-csi-driver/kubernetes"
-  version = "3.3.1"
+  source  = "github.com/convox/terraform-kubernetes-ebs-csi-driver?ref=48f5650f72684b581697f8831b3f5b60ea624092"
 
   ebs_csi_controller_image                   = "public.ecr.aws/ebs-csi-driver/aws-ebs-csi-driver"
   ebs_csi_driver_version                     = "v1.9.0"
   ebs_csi_controller_role_name               = "convox-ebs-csi-driver-controller"
   ebs_csi_controller_role_policy_name_prefix = "convox-ebs-csi-driver-policy"
-  oidc_url                                   = aws_iam_openid_connect_provider.cluster.url
+  csi_controller_tolerations = [
```

Comment:
this is based in https://github.com/kubernetes-sigs/aws-ebs-csi-driver/issues/758

`{ operator = "Exists", key = "CriticalAddonsOnly" }` will tolerate the taint `CriticalAddonsOnly` - it's a standard k8s taint.

`{ operator = "Exists", effect = "NoExecute", toleration_seconds = 300 }` - pods will tolerate any taint for at most 300 seconds - so it can detach any volumes or do any clean up needed.